### PR TITLE
Remove inline javascript onclicks #6961

### LINKED ISSUE #6961: Remove inline javascript onclicks
Many pages use inline scripting such as `onclick=...`.

### PR DESCRIPTION
**Pull request by thenaesh**
Fixes #6961 partially.

### CODE REVIEW

**Comment by thenaesh** [comment]
@wkurniawan07 @Neurrone @li-kai @whipermr5 Whoever is free/bored... please review. xD

File: src/main/webapp/js/instructor.es6 [hunk 1]
```diff
@@ -251,9 +251,11 @@ function bindStudentPhotoHoverLink(elements) {
         trigger: 'manual',
         placement: 'top',
         content() {
-            return '<a class="cursor-pointer" onclick="'
-                   + 'loadProfilePictureForHoverEvent($(this).closest(\'.popover\').siblings(\'.profile-pic-icon-hover\'))">'
-                   + 'View Photo</a>';
+            $('body').on('click', '.cursor-pointer', (event) => {
```
Comment (whipermr5) [inline src/main/webapp/js/instructor.es6]: Why not `$('.cursor-pointer').on('click')`?

**Comment by thenaesh** [comment]
@whipermr5 Done. I think there was a reason why I did it the other way, but I forgot the reason, so I changed it back. xD

**Comment by chowyb** [comment]
@thenaesh @whipermr5 from memory, using `$('.cursor-pointer').on('click')` (or similar method) has the problem that when adding in new elements with that class later (after the DOM has finished loading), these elements do not have the event attached, and thus will not function.

**Comment by thenaesh** [comment]
@chowyb That explains the test failure.

**Pull request merged**